config/upgrade/v0.15.0/upgrade.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: v0.15.0-upgrade
+  namespace: knative-eventing
+  labels:
+    eventing.knative.dev/release: devel
+spec:
+  template:
+    metadata:
+      annotations:
+        sidecar.istio.io/inject: "false"
+    spec:
+      serviceAccountName: eventing-controller
+      restartPolicy: Never
+      containers:
+      - name: upgrade-brokers
```

Comment:
perhaps the name is a bit too generic ? 

It's same as here:
https://github.com/knative/eventing/blob/master/config/upgrade/v0.14.0/upgrade.yaml#L17

:thinking:

Reply:
It's just the container, I'd rather leave it and not bake the version in there as well? It's already in the job.